content/en/cosign/verify.md

```diff
@@ -9,8 +9,24 @@ The general verification format with the `cosign verify` command is as follows.
 ```shell
 $ cosign verify --key <key path>|<key url>|<kms uri> <image uri> 
 ```
+## Keyless verification using OpenID Connect
 
-We'll use `user/demo` as our example image in the following commands. 
+We'll use `user/demo` as our example image in the following commands and keyless signing where appropriate.
+
+To verify a container image keylessly, use the following command:
```

Comment:
I think it would be good to have examples each for GitHub, Google, and Microsoft, but this is a good first pass for now

Reply:
@haydentherapper  do you have examples of using github, google, and microsoft?  Do you feel we need them?

Reply:
Google is https://accounts.google.com. Microsoft is https://login.microsoftonline.com, GitHub is https://github.com/login/oauth. 

I think adding them in would be good, we've gotten some questions about this. Some of the Cosign maintainers have chatted about making it easier in the future, but for now, this would be helpful to add

Reply:
OK, I will add them in.